packages/yoga/src/Dropdown/web/Dropdown.jsx

```diff
@@ -112,11 +140,40 @@ const Input = styled.input`
     line-height: ${dropdown.input.font.lineHeight}px;
     cursor: ${disabled ? 'not-allowed' : 'pointer'};
 
-    &, &::placeholder {
-      color: ${dropdown.input.font.color};
-      ${selected ? `color: ${dropdown.selected.input.font.color};` : ''}
-      ${disabled ? `color: ${dropdown.disabled.input.font.color};` : ''}
+    &:focus {
+      & ~ legend {
+        max-width: 1000px;
```

Comment:
These "hardcoded" `max-width`s are kinda tricky, what's the rationale behind them (so we can think on a good way to remove if possible)?

Reply:
I bring these changes from the Input component, there we have this [commit](https://github.com/Gympass/yoga/pull/164/commits/0897595f6dd6c2ef029cd741a7cfbf9b43f54180#diff-deb82c5065b3562f56a527453a2c76c3fd13479575decd49aca87caa56f3b224R59)
I'm not sure if this hardcoded max-width is necessary, I removed it and it keeps the expected behavior.

Reply:
Good to know, I wasn't aware of this, I'll gave it a look. Thanks!

Reply:
Just a quick update on this: I noticed that in fact a `max-width` is needed, since without it we'll have this behavior:
![image](https://user-images.githubusercontent.com/28108272/140511579-3446a2d8-c9dd-47c8-aaa9-275bf171bd42.png)

_But_, `1000px` is too hacky, what do you think about:
```suggestion
        max-width: max-content;
```

I tested with this and it works the same. You just have to update yours and I'll take care of the already existing Inputs 🌻